static/scss/faculty-members.scss

```
@@ -0,0 +1,111 @@
+// sass-lint:disable mixins-before-declarations
+.faculty-section {
+  margin: 0 0 20px;
+
+  .container {
+    h2 {
+      margin: 0 0 35px;
+
+      @include media-breakpoint-down(sm) {
+        margin: 0 0 27px;
+      }
+    }
+  }
+}
+
+.faculty-members {
+  padding: 0;
+  margin: 0 -22px;
+  list-style: none;
+  letter-spacing: -0.32em;
+
+  .course-card-description {
+    overflow-y: auto;
+    margin-bottom: 30px;
+  }
+
+  li {
+    @include media-breakpoint-down(md) {
+      width: calc(100% - 44px);
+    }
+
+    letter-spacing: normal;
+    display: inline-block;
+    vertical-align: top;
+    width: calc(50% - 44px);
+    margin: 0 22px 24px;
+
+    .member-card {
+      padding: 24px;
+      background: white;
+      border-bottom: 3px solid transparent;
+      box-shadow: 0 2px 4px 0 rgba(0,0,0,0.16);
+      overflow: hidden;
+
+      .member-info {
+        overflow: hidden;
+        margin-top: 7px;
+
+        @include media-breakpoint-down(md) {
+          margin-top: 20px;
+        }
+
+        h2 {
+          color: #000000;
+          font-size: 18px;
+          font-weight: 600;
+          letter-spacing: 0;
+          line-height: 22px;
+          margin: 0 0 5px;
+        }
+
+        p {
+          color: #333333;
+          font-size: 14px;
+          font-weight: 400;
+          letter-spacing: 0;
+          line-height: 22px;
+          margin-bottom: 0;
```

I don't think any of these styles are necessary besides the `font-size`. Please make an effort to reduce the amount of styling code you're adding to the codebase

```suggestion
          font-size: 14px;
```